Realm/Realm.Sync/Permissions/ManagementObjectStatus.cs

```diff
@@ -16,26 +16,29 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+using System;
+
 namespace Realms.Sync
 {
     /// <summary>
     /// The status of the management object as set by the server.
     /// </summary>
+    [Flags]
```

Comment:
These are flags only to enable: `user.GetPermissionOffers(ManagementObjectStatus.Success | ManagementObjectStatus.Error)` to get all offers in the success or error states. Perhaps I can make it a `params` argument and avoid the flags?